src/components/DependencyManager.tsx

```diff
@@ -167,73 +172,59 @@ const DependencyManager = ({ isOpen, onClose, onPipListUpdate = () => {} }) => {
     })();
   }, [depChanged]);
 
-  const runPipCommand = (args) => {
+  const runPipCommand = (args: string[]) => {
     setShellOutput('');
     setIsRunningShell(true);
-    const command = spawn(pythonKeys.python, ['-m', 'pip', ...args, '--disable-pip-version-check']);
+    // TODO: hope and pray pythonKeys is non-null
```

Comment:
It should be. Chainner would error before starting up if it was

Reply:
So actually, it turns out it IS null on the first render before it makes the request to get the keys.